impl/shim/shimTests/src/main/java/org/pentaho/big/data/impl/shim/tests/TestShimLoad.java

```diff
@@ -75,11 +75,13 @@ public TestShimLoad( MessageGetterFactory messageGetterFactory,
           messageGetter.getMessage( TEST_SHIM_LOAD_SHIM_LOADED_MESSAGE, activeConfigurationId ),
           ClusterRuntimeTestEntry.DocAnchor.SHIM_LOAD ) );
     } catch ( NoShimSpecifiedException e ) {
+      e.printStackTrace();
```

Comment:
Why printstacktrace? Not logger using? 
This place logs all the exceptions at the end of tests  - https://github.com/pentaho/big-data-plugin/blob/6db1566b1843ddb6b1e57e9372ab84a7be8ad5d5/kettle-plugins/common/ui/src/main/java/org/pentaho/big/data/plugins/common/ui/ClusterTestDialog.java#L209. Does it print exception when detailed level set?